src/main/resources/lib/samples/code.jelly

```diff
@@ -0,0 +1,31 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout">
+  <st:documentation>
+    <st:attribute name="code" />
+    <st:attribute name="file" />
+    <st:attribute name="language" />
+    <st:attribute name="executable" />
```

Comment:
what's this for?

Reply:
Ah that was a left-over from a branch this was based off of. Essentially it adds a button to the code block to enable you to execute whatever code is inside the block, so for example on the Notifications page you could show a notification by executing the code block.

I've removed it now, thanks for spotting :)